File: engine/b/src/main/java/net/crazyinvestor/engine_b/dto/BithumbWSTickerResponse.java
package net.crazyinvestor.engine_b.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public final class BithumbWSTickerResponse {
    private String symbol;
    private String tickType;
    private String date;
    private String time;
    private String openPrice;
    private String closePrice;
    private String lowPrice;
    private String highPrice;
    private String value;
    private String volume;
    private String sellVolume;
    private String buyVolume;
    private String prevClosePrice;
    private String chgRate;
    private String chgAmt;
    private String volumePower;
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/dto/response/BithumbWSTickerResponse.kt
package net.crazyinvestor.ticker_producer_engine.dto.response

class BithumbWSTickerResponse (
    val symbol: String?,
    val tickType: String?,
    val date: String?,
    val time: String?,
    val openPrice: String?,
    val closePrice: String?,
    val lowPrice: String?,
    val highPrice: String?,
    val value: String?,
    val volume: String?,
    val sellVolume: String?,
    val buyVolume: String?,
    val prevClosePrice: String?,
    val chgRate: String?,
    val chgAmt: String?,
    val volumePower: String?
){
    constructor(): this(null, null, null, null, null,null,
        null, null,null, null, null,null, null, null,
        null, null
    )
}